lang/java/reef-common/src/main/java/org/apache/reef/runtime/common/driver/evaluator/EvaluatorDescriptorImpl.java

```diff
@@ -80,4 +94,8 @@ public int getNumberOfCores() {
   public String getRuntimeName() {
     return this.runtimeName;
   }
+
+  public Map<String, String> getNodeLabelExpressions() {
+    return this.nodeLabelExpressions.get();
```

Comment:
Thank you for your comment! I will return `ImmutableMap<String, String>` for all getters.